src/main/java/Duke.java

```diff
@@ -1,10 +1,35 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        TaskManager taskManager = new TaskManager();
+        taskManager.greet();
+        Scanner line = new Scanner(System.in);
+        while(true) {
+            String[] input = line.nextLine().split(" ", 2);
```

Comment:
Perhaps can replace the magic literal "2" with a constant? That or maybe a comment about why you used 2 specifically will help.